tools/ci_connector_ops/ci_connector_ops/pipelines/commands/groups/connectors.py

```diff
@@ -33,11 +33,13 @@
 # HELPERS
 
 
-def validate_environment(is_local: bool, use_remote_secrets: bool):
+def validate_environment(is_local: bool, use_remote_secrets: bool, docker_registry: str):
     """Check if the required environment variables exist."""
     if is_local:
         if not (os.getcwd().endswith("/airbyte") and Path(".git").is_dir()):
             raise click.UsageError("You need to run this command from the airbyte repository root.")
+        if docker_registry == "airbyte":
+            raise click.UsageError("You should not use the airbyte docker hub registry when running locally. Please use a personal one.")
```

Comment:
Would we consider having a separate testing repository that everyone could use, instead of each person having to set up a personal one? 

If using a personal, do you have to do anything to your personal registry beforehand to make this work?

Reply:
If you have a dockerhub account you have a registry by default whose name is your username, so no extra setup is required.
I agree that a separate global testing registry could be more convenient.

Reply:
Yeah, I'm thinking that there are some people who probably prefer not to have 300+ images on their public repo, especially if they leave the company and don't want to delete them all, or have other ones that become hard to find in the mix. I was worried about the public/private distinction, but looks like you can have as many public images as you like (which is the default)